php-checks/src/main/java/org/sonar/php/checks/formatting/IndentationCheck.java

```diff
@@ -208,21 +199,36 @@ private void checkClosingParenthesisLocation(Tree lastArgument, SyntaxToken clos
     }
   }
 
-  private static boolean isCorrectlyIndented(int expectedColumn, List<? extends Tree> items) {
+  private static boolean areIncorrectlyIndented(int expectedColumn, List<? extends Tree> items) {
     for (Tree item : items) {
-      if (((PHPTree) item).getFirstToken().column() != expectedColumn) {
-        return false;
+      if (isIncorrectlyIndented(expectedColumn, (PHPTree) item)) {
+        return true;
       }
     }
-    return true;
+    return false;
+  }
+
+  private static boolean isIncorrectlyIndented(int expectedColumn, PHPTree item) {
+    if (isNamedArgument(item)) {
+      PHPTree parent = (PHPTree) item.getParent();
+      SyntaxToken firstToken = parent.getFirstToken();
+      return firstToken.column() != expectedColumn;
+    } else {
+      return item.getFirstToken().column() != expectedColumn;
+    }
+  }
+
+  private static boolean isNamedArgument(PHPTree item) {
+    PHPTree parent = (PHPTree) item.getParent();
+    return parent!=null && parent.is(Kind.CALL_ARGUMENT) && ((CallArgumentTree) parent).name() != null;
```

Comment:
How come you added back the parent null check eventually?
Also maybe we can gain one call to the `getParent()` method by doing the following, what do you think?
```suggestion
  private static boolean isIncorrectlyIndented(int expectedColumn, PHPTree item) {
    PHPTree parent = (PHPTree) item.getParent();
    if (isNamedArgument(parent)) {
      SyntaxToken firstToken = parent.getFirstToken();
      return firstToken.column() != expectedColumn;
    } else {
      return item.getFirstToken().column() != expectedColumn;
    }
  }

  private static boolean isNamedArgument(PHPTree item) {
    return item!=null && item.is(Kind.CALL_ARGUMENT) && ((CallArgumentTree) item).name() != null;
```